patches/path.patch

```diff
@@ -0,0 +1,36 @@
+--- ./github.com/jipanyang/gnxi/utils/xpath/path.go
++++ ./github.com/jipanyang/gnxi/utils/xpath/path.go
+@@ -18,6 +18,7 @@ package xpath
+ 
+ import (
+ 	"fmt"
++	"strings"
+ 
+ 	pb "github.com/openconfig/gnmi/proto/gnmi"
+ )
+@@ -45,9 +46,17 @@ func ToGNMIPath(xpath string) (*pb.Path, error) {
+ 		return nil, err
+ 	}
+ 	var pbPathElements []*pb.PathElem
+-	for _, elem := range xpathElements {
++	origin := ""
++	for i, elem := range xpathElements {
+ 		switch v := elem.(type) {
+ 		case string:
++			if i == 0 {
++				firstElement := strings.SplitN(v, ":", 2)
```

Comment:
>SplitN [](http://example.com/codeflow?start=29&length=6)

Add some comment: why we need this patch?

Reply:
Support origin for gnmi client.